### PR TITLE
Separate EntityControlMenu as a common component

### LINKED ISSUE #431: (React / EntityList) Entity operation menu item
Both the entity list and the entry list have entity operation menus, but the items are different.
I think it's better to make them the same.

- EntityList
image

- EntryList
image

### PR DESCRIPTION
**Pull request by syucream**
To resolve https://github.com/dmm-com/airone/issues/431, and experimentally implement https://github.com/dmm-com/airone/issues/416 and https://github.com/dmm-com/airone/issues/430

It has common component on entity list page and entry list page.

- entity list page
image

- entry list page
image
 
with notifying asynchronous events on Snackbar for experiment.
image

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -11,7 +12,10 @@ const App: FC = () => {
   return (
     <ThemeProvider theme={theme}>
       <ErrorHandler>
-        <AppRouter />
+        {/* TODO does it unable to apply custom styles? reimplement it ourselves? */}
```
Comment (syucream) [inline frontend/src/App.tsx:15]: It depends on https://github.com/iamhosseindhv/notistack to make easy to notify events on Snackbar entirely. But, unfortunately it seems not to support custom styles with MUI-v5. https://smartdevpreneur.com/customizing-notistack-with-material-ui-snackbar/

I'm considering if its better to reimplement similar stack ourselves or resolve the problem with notistack 🤔

Comment (hinashi) [reply frontend/src/App.tsx:15]: So easy to make and great!
I think the design is no problem for now.

**Review by hinashi — APPROVED** [review]
LGTM!

**Review by userlocalhost — APPROVED** [review]
Thank you very much for your great work!!

**Pull request merged**